secboot/secboot_tpm_test.go

```diff
@@ -494,6 +499,26 @@ func (s *secbootSuite) TestSealKey(c *C) {
 			mockEFI[0] = "/does/not/exist"
 		}
 
+		// create a snap file
+		foo := filepath.Join(tmpDir, "foo.snap")
+		var foosnap snap.Container
+		if tc.badSnapFile {
+			err := ioutil.WriteFile(foo, nil, 0644)
+			c.Assert(err, IsNil)
+		} else {
+			snapdir := c.MkDir()
+			err := os.MkdirAll(filepath.Join(snapdir, "meta"), 0755)
```

Comment:
A little bit of duplication could be saved via something like:
```
snapYamlPath := filepath.Join(c.MkDir(), "meta/snap.yaml")
err := os.MkdirAll(filepath.Dir(snapYamlPath), 0755)
c.Assert(err, IsNil)
err = ioutil.WriteFile(filepath.Join(snapYamlPath, []byte("name: foo"), 0644)
c.Assert(err, IsNil)
...
```

Reply:
Did that and renamed variables to something a bit more descriptive.